Podfile

```diff
@@ -26,8 +26,8 @@ target "eXo" do
 end
 
 target "share-extension" do
-    pod 'UICKeyChainStore', '~> 2.1'
-    pod 'HTMLKit', '~> 3.1'
+    pod 'UICKeyChainStore'
+    pod 'HTMLKit', '~> 3.1.0'
```

Comment:
We need to use a newer version of HTMLKit

Reply:
Yes the last changes :
pod 'UICKeyChainStore'
pod 'HTMLKit', '~> 3.1.0'